src/Runner.Common/JobServerQueue.cs

```diff
@@ -819,6 +820,18 @@ private List<TimelineRecord> MergeTimelineRecords(List<TimelineRecord> timelineR
             return mergedRecords;
         }
 
+        private void removeDuplicatedIssues(TimelineRecord timelineRecord)
```

Comment:
nit:  C# naming standards are PascalCase, not camelCase.

Reply:
Ah right, golang went too deep :D